build.sbt

```diff
@@ -44,6 +68,5 @@ lazy val azure = project
       // Azure doesn't pull in libs like netty-transport-native-kqueue,
       // netty-transport-native-unix-common and netty-transport-native-epoll.
       // Keep nettyVersion in sync with the version that Azure pulls in.
-      "io.netty" % "netty-all" % "4.1.38.Final",
```

Comment:
How bad is not including this? I found it tedious to keep it in sync with the Azure version.

Reply:
Hmm, I'm not sure about the exact effects. It may actually be better to remove it here, now that `netty-all` is also included any way from here: https://github.com/slamdata/quasar/blob/79e9a44297d0a5aca57c4e9d94eaeb5b9a780f68/build.sbt#L281

Let's remove it here. Don't forget to remove the comments above too.